### DISCUSSION TITLE
Pop up claims another app is controlling Brightness

### DISCUSSION THREAD

**Discussion by ChiefS9** [MonitorControl/MonitorControl#965] [Q&A] [answered]
Hi. 

I am on MacOS Monterey V 12.1. 

I have recently had a pop up from Monitor control claiming that another app controls my monitor brightness. It gave me the option of closing the app or disabling the gamma controls in system preferences. When I received this pop up I tried to figure out which app was causing this issue and I was unable to discern any app on my mac which should be conflicting with Monitor control. 

I eventually clicked on quitting the app but this did not do anything. This has piqued my interest as I have no idea why this occurred. As I cannot think of any software which should be influencing my monitor in this way I would like to be sure that nothing is amiss. 

I was wondering if anyone else had encountered this problem? If so would there be anything they could share with me that would be of help. 

Thanks

**Comment by waydabber** [comment, marked as answer]
Hi, apps like f.lux or Lunar can cause this - apps that changes the gamma table. If this happened only once and not again since, then it might be a false alarm (maybe the display driver did reset or something).

**Reply by ChiefS9** [reply]
Sorry for the lack of response. It has not occured again and as such I feel it must have just been a one time thing. Thank you for your prompt response and maintaining the software.
